extensions/wikia/UserProfilePageV3/templates/UserProfilePage_renderLightbox.php

```diff
@@ -81,36 +82,46 @@
 					</select>
 				</div>
 				<div class="input-group">
-					<label for="occupation"><?= wfMsg('user-identity-box-about-occupation'); ?></label>
-					<input type="text" name="occupation" value="<?= $user['occupation']; ?>" maxlength="<?= $charLimits['occupation']; ?>" />
+					<label for="occupation"><?= wfMessage( 'user-identity-box-about-occupation' )->escaped(); ?></label>
+					<input type="text" id="occupation" name="occupation" value="<?= $user['occupation']; ?>" maxlength="<?= $charLimits['occupation']; ?>"/>
+				</div>
+				<div class="input-group">
+					<label for="gender"><?= wfMessage( 'user-identity-box-about-gender' )->escaped(); ?></label>
+					<input type="text" id="gender" name="gender" value="<?= $user['gender']; ?>" maxlength="<?= $charLimits['gender']; ?>"/>
 				</div>
 				<div class="input-group">
-					<label for="gender"><?= wfMsg('user-identity-box-about-gender'); ?></label>
-					<input type="text" name="gender" value="<?= $user['gender']; ?>" maxlength="<?= $charLimits['gender']; ?>" />
+					<label for="website"><?= wfMessage( 'user-identity-box-about-website' )->escaped(); ?></label>
+					<input type="text" id="website" name="website" value="<?= $user['website']; ?>">
 				</div>
 				<div class="input-group">
-					<label for="website"><?= wfMsg('user-identity-box-about-website'); ?></label>
-					<input type="text" name="website" value="<?= $user['website']; ?>">
+					<label for="twitter"><?= wfMessage( 'user-identity-box-about-tweet' )->escaped(); ?></label>
+					<label for="twitter" class="tweet-at">twitter.com/</label>
+					<input type="text" id="twitter" name="twitter" value="<?= $user['twitter']; ?>">
 				</div>
 				<div class="input-group">
-					<label for="twitter"><?= wfMsg('user-identity-box-about-tweet'); ?></label>
-					<span class="tweet-at">@</span>
-					<input type="text" name="twitter" value="<?= $user['twitter']; ?>">
+					<label for="fbPage"><?= wfMessage( 'user-identity-box-about-fb-page' )->escaped(); ?></label>
+					<label for="fbPage" class="fb-start">facebook.com/</label>
+					<input type="text" id="fbPage" name="fbPage" value="<?= $user['fbPage']; ?>">
```

Comment:
Thanks @jsutterfield for your change!
@Grunny great explanation. 
What do you think about creating best practises for escaping?
Having nice rule could increase our efficiency in this area. For example when displaying data, sanitizing should happen as late as possible - in our case it's template. Quickly I can enumerate some advantages of this approach:
* we won't escape data 2 times by mistake
* we don't need to go back and check if data was escpaed already or not
* we could use proper sanitizing methods for different places
* data that is sent though API is not escaped in controller and by API again

The same rule probably can be applied for data that is going to be saved (in model). What do you think about it?